venter/admin.py

```diff
@@ -1,9 +1,13 @@
 """Admin models for venter."""
 from django.contrib import admin
+from django.core.mail import send_mass_mail
+
+from backend import settings_base
```

Comment:
`from django.conf import settings` instead of importing `settings_base`, so we can override settings in production